### PR TITLE
Dat 2538 turn on MoM o 10% of new created wikis

### PR DESCRIPTION
**Pull request by RafLeszczynski**
DO NOT MERGE AFTER QA

### CODE REVIEW

File: extensions/wikia/CreateNewWiki/CreateWiki.php [hunk 1]
```diff
@@ -988,6 +988,7 @@ private function setWFVariables() {
 		$this->mWFSettingVars['wgEnableSectionEdit']      = true;
 		$this->mWFSettingVars['wgEnableSwiftFileBackend'] = true;
 		$this->mWFSettingVars['wgOasisLoadCommonCSS']     = true;
+		$this->mWFSettingVars['wgEnableNjordExt']         = intval( $this->mNewWiki->city_id ) % 10 = 1 ? true : false;
```
Comment (SebastianMarzjan) [inline extensions/wikia/CreateNewWiki/CreateWiki.php]: Did you mean === instead of = here?

**Comment by SebastianMarzjan** [comment]
One thing from me, rest LG

File: extensions/wikia/CreateNewWiki/CreateWiki.php (hunk 1 above)
Comment (RafLeszczynski) [inline extensions/wikia/CreateNewWiki/CreateWiki.php]: yes of course my mistake

File: extensions/wikia/CreateNewWiki/FinishCreateWikiController.class.php [hunk 2]
```diff
@@ -105,4 +111,19 @@ public function FinishCreate() {
 		$wgOut->redirect($mainPage.'?wiki-welcome=1');
 	}
 
+	/**
+	 *
+	 * @param $mianPageTitle string
+	 * @param $mainPageArticle Article
+	 */
+	private function initHeroModule( $mianPageTitle, $mainPageArticle ) {
```
Comment (idradm) [inline extensions/wikia/CreateNewWiki/FinishCreateWikiController.class.php]: Frist params name is misspelled, what about adding type hinting for article param

File: extensions/wikia/CreateNewWiki/FinishCreateWikiController.class.php [hunk 3]
```diff
@@ -84,17 +84,23 @@ public function FinishCreate() {
...
+					global $wgParser;
+					$mainPageText = $mainArticle->getRawText();
+					$matches = array();
+					$description = $this->params['wikiDescription'];
+					if(preg_match('/={2,3}[^=]+={2,3}/', $mainPageText, $matches)) {
+						$newSectionTitle = str_replace('Wiki', $wgSitename, $matches[0]);
+						$description = "{$newSectionTitle}\n{$description}";
+					}
+					$newMainPageText = $wgParser->replaceSection( $mainPageText, 1, $description );
+					$mainArticle->doEdit($newMainPageText, '');
```
Comment (idradm) [inline extensions/wikia/CreateNewWiki/FinishCreateWikiController.class.php]: Maybe this can be refactor, as this and initHeroModule does similiar thing

**Comment by idradm** [comment]
LGTM

**Comment by idradm** [comment]
:shipit:

**Pull request merged**